ability_item_usage_pudge.lua

```diff
@@ -130,10 +130,10 @@ Consider[1] = function()
                 enemies[1]:GetExtrapolatedLocation(GetUnitToUnitDistance(npcBot, enemies[1]) / hookSpeed)
         end
         do
-            local ally = AbilityExtensions:GetNearbyNonIllusionHeroes(npcBot, range, false, BOT_MODE_NONE):Filter(allies
-                , function(t)
+            local ally = AbilityExtensions:GetNearbyNonIllusionHeroes(npcBot, range, false, BOT_MODE_NONE):
+                Filter(function(t)
                 return t:IsStunned() or t:IsRooted()
```

Comment:
Does this line need an extra indent?

Reply:
Yeah, I think an indent is more correct. Changed.